TicTacToe/TicTacToe.cs

```diff
@@ -25,55 +26,118 @@ public static void Main()
             Console.ReadLine();
         }
 
+        // get row and column input from user
         public static void GetInput()
         {
             Console.WriteLine("Player " + playerTurn);
             Console.WriteLine("Enter Row:");
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+            PlaceMark(row, column);
         }
 
+
+        // place mark on the board in appropriate location, check for win, check for tie, and continue to next player move if neither
         public static void PlaceMark(int row, int column)
         {
         // your code goes here
+        board[row][column] = playerTurn;
+        bool winner = CheckForWin();
+        bool tie = CheckForTie();
+        System.Console.WriteLine(winner);
+            if (winner)
+            {
+                DrawBoard();
+                System.Console.WriteLine("Player " + playerTurn + " Won!");
+            }
+            else if (tie)
+            {
+                DrawBoard();
+                System.Console.WriteLine("It was a tie!");
+            }
+            else
+            {
+                playerTurn = (playerTurn == "X") ? "O" : "X";
+            }
+
         }
 
+
+        // method to check for all types of wins
         public static bool CheckForWin()
         {
             // your code goes here
-
+            if (HorizontalWin() || VerticalWin() || DiagonalWin())
+            {
+                return true;
+            }
             return false;
         }
 
+
+        // method to check for a tie
         public static bool CheckForTie()
         {
             // your code goes here
-
-            return false;
+            if ((board[0][0] != " ") && (board[0][1] != " ") && (board[0][2] != " ") && (board[1][0] != " ") && (board[1][1] != " ")
```

Comment:
This method should probably be called CheckForUnplayedSpots. This method will return false as soon as the last available person plays in the last spot. It is only after you _also_ check there is no Win that this method can be used to confirm there was a tie.